docs/guide/tutorial-3-appearance.md

```diff
@@ -70,7 +70,7 @@ The aim of this subsection is to apply a locally available texture to the ball.
 A texture on a rolling object can help to appreciate its movement.
 
 > **Hands-on #4**: add a locally available texture to the ball.
-1. Download the texture available [here](https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/red_brick_wall.jpg) and save it to your disk.
+1. Download the texture available [here](https://raw.githubusercontent.com/cyberbotics/webots/R2022a/projects/default/worlds/textures/red_brick_wall.jpg) and save it to your disk.
```

Comment:
```suggestion
1. Download the texture available [here](https://raw.githubusercontent.com/cyberbotics/webots/{{ webots.version.full }}/projects/default/worlds/textures/red_brick_wall.jpg) and save it to your disk.
```
Maybe it is better to use variables here

Reply:
It seems that there are other places where this variable could be used.